src/main/java/com/myproject/todo/todo/model/Todo.java

```diff
@@ -0,0 +1,45 @@
+package com.myproject.todo.todo.model;
+
+import java.util.Date;
+
+public class Todo {
+	
+	private Integer id;
+	private Date date; 
+	private String task;
+	public Integer getId() {
+		return id;
+	}
+	public void setId(int id) {
```

Comment:
id if autogenerated, should have the proper annotations. and should be string imo.

Reply:
generated by the service